content/doc/pipeline/tour/getting-started.adoc

```diff
@@ -24,7 +24,7 @@ For this tour, you will require:
 
 . http://mirrors.jenkins.io/war-stable/latest/jenkins.war[Download Jenkins].
 . Open up a terminal in the download directory.
-. Run `java -jar jenkins.war --httpPort=8080`.
+. Run `java -jar jenkins.war --httpPort=8080 --httpListenAddress='127.0.0.1'`.
 . Browse to `http://localhost:8080`.
```

Comment:
Probably the risks aren't too high since Jenkins starts up locked, and requires the user to enter token only available on the local machine.  From that point, it guess it's possible that someone on public wifi, where machines are open to all others, could decide to run the tutorial and start pulling in private source repos.

In any case, we shouldn't assume IPv4 — the listen address should be `localhost`.  This would also makes it consistent with the `http://localhost:8080` URL in the line below.

The quotes around the address are also unnecessary.

But also note that there are numerous tutorials on this site which do similar.  So perhaps just giving a warning in the tutorial may be more useful, rather than restricting people from the more likely scenario of sharing their test Jenkins setup with other people in their local network.

Reply:
I used `netstat -tnpl` to check bind port, it was 0.0.0.0
Yes, i agree, that it isn't critical bug. Also if you use it on public server, localhost bind is bad